packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -190,8 +205,12 @@ export class WaferMap extends FoundationElement {
         if (this.validity.invalidDiesTableSchema) {
             return;
         }
+        this.renderer = this.diesTable === undefined
```

Comment:
We are using `waferMap.diesTable === undefined` pretty often now as a signal for the experimental renderer. CAn we add an `@internal` method like `waferMap.isExperimentalRenderer()` to do that check. We should replace all the spots we are doing that check with the helper method.